Makefile

```diff
@@ -410,7 +410,7 @@ define generate
 endef
 
 define manifest
-	$(call controller-gen,crd:trivialVersions=true rbac:roleName=manager-role webhook paths="./..." output:crd:artifacts:config=config/crd/bases)
+	$(call controller-gen,crd paths="./..." output:crd:artifacts:config=chart/k8gb/templates/)
```

Comment:
is crd version v1 is the default in `sigs.k8s.io/controller-tools/cmd/controller-gen@v0.5.0` ? I recall explicit version statement in the first version of this PR

Reply:
yes, it is. Since v1beta1 is deprecated quite a while ago. We can have it explicit though